src/main/java/org/sonarsource/sonarlint/ls/connected/ProjectBindingManager.java

```diff
@@ -307,21 +313,28 @@ public void updateAllBindings() {
     // Clear cached bindings to force rebind during next analysis
     folderBindingCache.clear();
     fileBindingCache.clear();
-
+    List<String> failedServerIds = new ArrayList<>();
     // Start by updating all engines that are already started and cached
-    connectedEngineCacheByServerId.entrySet().forEach(entry -> {
-      String serverId = entry.getKey();
+    connectedEngineCacheByServerId.forEach((serverId, value) -> {
       ServerConfiguration serverConfiguration = createServerConfiguration(serverId);
       if (serverConfiguration != null) {
-        entry.getValue().ifPresent(engine -> updateGlobalStorageAndLogResults(serverConfiguration, engine));
+        try {
+          value.ifPresent(engine -> updateGlobalStorageAndLogResults(serverConfiguration, engine, failedServerIds, serverId));
+        } catch (Exception e) {
+
+        }
       }
     });
     Map<String, Set<String>> updatedProjectsByServer = new HashMap<>();
     updateBindingIfNecessary(null, updatedProjectsByServer);
 
     foldersManager.getAll().forEach(f -> updateBindingIfNecessary(f, updatedProjectsByServer));
-
-    client.showMessage(new MessageParams(MessageType.Info, "All SonarLint bindings succesfully updated"));
+    if (failedServerIds.isEmpty()) {
+      client.showMessage(new MessageParams(MessageType.Info, "All SonarLint bindings succesfully updated"));
+    } else {
+      String message = String.join(", ", failedServerIds);
+      client.showMessage(new MessageParams(MessageType.Error, "Binding update failed for the following servers: " + message));
```

Comment:
Same as above, give some hints to the user to know what's going wrong.

Reply:
Removed.